games-arcade/wormux/wormux-master.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Very nice game"
+DESCRIPTION="Arcade game based on Worms"
+HOMEPAGE="http://wormux.org"
+COPYRIGHT="<a-team>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/a-team/wormux/archive/master.zip"
```

Comment:
CHECKSUM missing

Reply:
Check for the order and for the reuired blocks: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines